spec/controllers/games_controller_spec.rb

```diff
@@ -138,7 +138,7 @@
         it 'successful' do
           allow_any_instance_of(GameBuilderService).to receive(:build).and_return(true)
           allow_any_instance_of(GameBuilderService).to receive(:game).and_return(double(Game, subscribe: true, id: 1, 'private_game?' => false))
-          post :create, {format: :json, step: 4, game: {poster: 'data:image/png;base64,iisudfhisfh778'}}
+          post :create, {format: :json, step: 4, game: {poster: Rack::Test::UploadedFile.new(Rails.root.join('spec','support', 'files', 'image.png'), 'image/png')}}
```

Comment:
Line is too long. [164/80]<br>Redundant curly braces around a hash parameter.<br>Space missing after comma.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.